packages/mds-compliance/api.ts

```diff
@@ -122,7 +122,7 @@ function api(app: express.Express): express.Express {
             .includes(policy.policy_id)
         ) {
           const [geographies, deviceRecords] = await Promise.all([
-            db.readGeographies(),
+            db.readGeographies() as Promise<Geography[]>,
```

Comment:
I am surprised that `readGeometries` doesn't actually return `Promise<Geography[]>` already. What does it return?